mmcv/transforms/base.py

```diff
@@ -12,12 +12,10 @@ def __call__(self, results: Dict) -> Dict:
     @abstractmethod
     def transform(self, results: Dict) -> Dict:
         """The transform function. All subclass of BaseTransform should
-        override this method.
-
-        This function takes the result dict as the input, and can add new
-        items to the dict or modify existing items in the dict. And the result
-        dict will be returned in the end, which allows to concate multiple
-        transforms into a pipeline.
+        override this method. This function takes the result dict as the input,
```

Comment:
We should not modify the format here. The original format is fine. The summary of the function should be simple and in 1-2 sentences. Then the description can be longer.